app/controllers/api/v1/widget/messages_controller.rb

```diff
@@ -48,7 +48,8 @@ def set_conversation
   def message_finder_params
     {
       filter_internal_messages: true,
-      before: permitted_params[:before]
+      before: permitted_params[:before],
+      after: permitted_params[:after]
```

Comment:
Note: Specs for param `after` is already added https://github.com/chatwoot/chatwoot/blob/develop/spec/finders/message_finder_spec.rb#L51